litellm/proxy/proxy_server.py

```diff
@@ -5157,7 +5283,7 @@ async def run_thread(
         api_base = hidden_params.get("api_base", None) or ""
 
         fastapi_response.headers.update(
-            ProxyBaseLLMRequestProcessing.get_custom_headers(
```

Comment:
can you not undo this change, we recently moved this to ProxyBaseLLMRequestProcessing